### DISCUSSION TITLE
Upgrade problem

### DISCUSSION THREAD

**Discussion by komir2004** [akeeba/panopticon#392] [Q&A (Community Support)] [answered]
After last upgrade to v.1.0.6
get error 
`Call to a member function getRecommendedSupportedBranch() on null
[View template: ROOT/tmp/compiled_templates/cbd90bb774e2c7613fda3e37ec6ae7a23f46578d.php line 48, compiled from ROOT/ViewTemplates/Main/php_warnings.blade.php]`

Can you suggest what to do? 
I am using two factor authentication, maybe that caused a problem with upgrade

![Call-to-a-member-function-php_warn](https://github.com/akeeba/panopticon/assets/17553018/01f979b4-bea7-477b-99ed-7fdbf41cb7af)

**Comment by nikosdion** [comment]
Have you tried deleting the `tmp/compiled_templates` folder?

**Comment by komir2004** [comment]
I have not attempted to delete the folder "compiled_templates". I just deleted it now and refreshed Panoticon, but the error still occurs and the folder is automatically recreated.

**Reply by nikosdion** [reply, marked as answer]
Please replace the `ViewTemplates/Main/php_warnings.blade.php` with the latest version from the repository.

**Reply by komir2004** [reply]
I appreciate your assistance, and I'm glad to report that the problem has been resolved.

**Reply by nikosdion** [reply]
Thank you for the feedback! I am trying to finish up a couple more things and release an update over the next couple of days.

**Comment by komir2004** [comment]
Unfortunately, when I try to enter the some web sites, I get an error message that says:

`Invalid JSON data returned from the server: ‘’.
[View template: ROOT/tmp/compiled_templates/a4b31ce4609a6b78b9a571b11505bf39f516c18f.php line 476, compiled from ROOT/ViewTemplates/Sites/item_backup.blade.php]`

I tried to replace the `/ViewTemplates/Sites/item_backup.blade.php` file with the latest version from the repository, but it did not help.

**Reply by nikosdion** [reply]
Read the message again. It tells you that your (Joomla) site's server returned no response.
